pyprep/find_noisy_channels.py

```diff
@@ -377,321 +376,22 @@ def find_bad_by_ransac(
-                chn_pos, chn_pos_good, good_chn_labs, data, sample
-            )
-
-        # Form median from all predictions
-        ransac_eeg = np.median(eeg_predictions, axis=-1, overwrite_input=True)
-        return ransac_eeg
-
-    def get_ransac_pred(self, chn_pos, chn_pos_good, good_chn_labs, data, sample):
-        """Perform RANSAC prediction.
-
-        Parameters
-        ----------
-        chn_pos : ndarray
-            3-D coordinates of the electrode position
-        chn_pos_good : ndarray
-            3-D coordinates of all the channels not detected noisy so far
-        good_chn_labs : array_like
-            channel labels for the ch_pos_good channels
-        data : ndarray
-            2-D EEG data
-        sample : int
-            the current RANSAC sample number
-
-        Returns
-        -------
-        ransac_pred : ndarray
-            Single RANSAC prediction
-
-        """
-        # Get the random channel selection for the current sample
-        reconstr_idx = self.random_ch_picks[sample]
-
-        # Get positions and according labels
-        reconstr_labels = good_chn_labs[reconstr_idx]
-        reconstr_pos = chn_pos_good[reconstr_idx, :]
-
-        # Map the labels to their indices within the complete data
-        # Do not use mne.pick_channels, because it will return a sorted list.
-        reconstr_picks = [
-            list(self.ch_names_new).index(chn_lab) for chn_lab in reconstr_labels
-        ]
-
-        # Interpolate
-        interpol_mat = _make_interpolation_matrix(reconstr_pos, chn_pos)
-        ransac_pred = np.matmul(interpol_mat, data[reconstr_picks, :])
-        return ransac_pred
-
-    def ransac_correlations(
-        self,
-        chans_to_predict,
-        chn_pos,
-        chn_pos_good,
-        good_chn_labs,
-        data,
-        n_samples,
-        n,
-        w_correlation,
-    ):
-        """Get correlations of channels to their ransac predicted values.
-
-        Parameters
-        ----------
-        chans_to_predict: list of int
-            Indexes of the channels to predict.
-        chn_pos : ndarray
-            3-D coordinates of the electrode positions to predict
-        chn_pos_good : ndarray
-            3-D coordinates of all the channels not detected noisy so far
-        good_chn_labs : array_like
-            channel labels for the ch_pos_good channels
-        data : ndarray
-            2-D EEG data
-        n_samples : int
-            Number of samples used for computation of ransac.
-        n : int
-            Number of frames/samples of each window.
-        w_correlation: int
-            Number of windows.
-
-        Returns
-        -------
-        channel_correlations : ndarray
-            correlations of the given channels to their ransac predicted values.
+        """Find bad channels by the ransac method.
 
+        In essence a wrapper of find_bad_by_ransac function for the NoisyChannels Class.
```

Comment:
This is something that users see when they inspect this public method through `help()` or similar --> so please add the usual "parameters", "returns", etc. sections. 

It's okay to mention that this is a wrapper around another function, but I suggest to reword it:

```suggestion
        This method is a wrapper of the ``find_bad_by_ransac`` function
        from the ``ransac`` module.
```